optimum/intel/__init__.py

```diff
@@ -17,3 +17,4 @@
 from .neural_compressor.pruning import IncPruner, IncPruningMode
 from .neural_compressor.quantization import IncQuantizationMode, IncQuantizer
 from .neural_compressor.trainer import IncTrainer
+from .version import __version__
```

Comment:
Since `optimum-intel` is now a fully-fledged python package, I included the version here to make it easier to build the docs (otherwise I have to manually specify the version which is annoying)